### DISCUSSION TITLE
Relations aren't working right. I guess

### DISCUSSION THREAD

**Discussion by KingImoh** [pocketbase/pocketbase#1685] [Q&A]
Why doesn't the related collection update when the collection with the relations has a new record. Am i doing something wrong.
![image](https://user-images.githubusercontent.com/70730188/214573624-e372be44-dda6-4dd1-83a6-99f65c63c744.png)
![image](https://user-images.githubusercontent.com/70730188/214573909-ffaea738-c117-4bba-bb4a-fcd12fc0274c.png)

**Comment by ganigeorgiev** [comment]
I'm not sure that I understand what do you mean. Which relation doesn't update?

The admin UI is not binded to the realtime events at the moment, and you have to click the refresh btn or reload the page to see the new records, if that's what you are asking.
